src/main/java/dylanbot/Task.java

```diff
@@ -0,0 +1,57 @@
+package dylanbot;
+
+/**
+ * Represents a general Task
+ */
+public class Task {
+
+    private String type;
+    private String desc;
+    private boolean completed;
+
+    /**
+     * Creates a new Task with the given type and description
+     *
+     * @param type The specified type
+     * @param desc The specified description
+     */
+    public Task(String type, String desc) {
+        this.type = type;
```

Comment:
Love that you have a type attribute so that you can change the value easily if needed! 💯